glue_jupyter/bqplot/histogram/layer_artist.py

```diff
@@ -1,148 +0,0 @@
-import numpy as np
```

Comment:
Even though the new code is much more concise, could you maybe keep the separation between files of the layer artist and viewer and so on? (just as it is a pretty common pattern I have been using for viewers so I find it easier to navigate)